### DISCUSSION TITLE
How should I go about using pbcopy with vim-julia-cell?

### DISCUSSION THREAD

**Discussion by ThomasChaffey** [mroavi/vim-julia-cell#3] [Q&A]
I just installed on OSX.

Executing the whole file works like a dream.

Executing a single cell gives me the error "Could not find xclip or xsel executable".

Is there a config option to use pbcopy?

Thanks!

**Comment by mroavi** [comment]
I don't have OSX to test this. I read that `xclip` can be installed on Mac with brew:

```
brew install xclip
```

Does this solve the problem?

**Comment by mroavi** [comment]
Let me know if this works. I noticed I have to update the readme. `pbcopy` is not supported as I don't have a way to test on Mac.

**Comment by mroavi** [comment]
I can however make a quick change to the source code to try to support `pbcopy`. I would need you to test for me, however.

**Reply by keita-sunada** [reply]
Hi. I also try to use vim-julia-cell on mac OS. 
I encountered the same error "Could not find xclip or xsel executable", so  installed xclip via homebrew based on your suggestion. 
After that I success to send cells to REPL. 
However then the error ``Can't open display: (null)'' emerged, and is not still resolved.  Maybe this is related to issue #1 and not related to this discussion.

**Reply by mroavi** [reply]
Hi Keita. Unfortunately, I don't have a mac setup to test this. If you find a solution to this problem I would appreciate it if you send a PR. I'm not using this plugin anymore.

By the way, I noticed that vim-slime added the option to run cells with custom delimiters (see [here](https://github.com/jpalardy/vim-slime#advanced-configuration)). This means that this plugin should not be needed anymore. Let me know if this helps.
